ports/nrf/boards/arduino_nano_33_ble/mpconfigboard.h

```diff
@@ -0,0 +1,76 @@
+/*
+ * This file is part of the MicroPython project, http://micropython.org/
+ * The MIT License (MIT)
+ * Copyright (c) 2022 Arduino SA
+ */
+
+#define MICROPY_HW_BOARD_NAME       "Arduino Nano 33 BLE SENSE"
```

Comment:
Does the board name include the word "SENSE"?  From the packaging I see just "Arduino Nano 33 BLE".  Also you called the board directory `arduino_nano_33_ble`.

Reply:
They are both the exact same board, but the "SENSE" one has all the extra sensors populated. But technically you're right, so I renamed the board, fixed links, etc...and added the right image here:

https://github.com/micropython/micropython-media/pull/23